apiary.apib

```diff
@@ -3,8 +3,8 @@ FORMAT: X-1A
 HOST: https://co2.eaternity.ch/api/
 
 # Eaternity Database API
-The Eaternity Database API is a Web Service that calculates the Climate Score (CO₂eq), the Vita Score and other environmental indicators of recipes, restaurants and ingredients dynamically. It includes daily changing information on origin,
-seasonality and other factors in the calculation ([More Informations on the indicators](http://www.eaternity.org/foodprint/)). It further delivers reports with sophisticated and easy to understand statistics, graphs and analysis ([More Informations on the reports](http://www.eaternity.org/app/report)).
+The Eaternity Database API is a Web Service that calculates the Climate Score (CO₂eq), the Vita Score and other environmental footprints of recipes, restaurants and ingredients dynamically. It includes daily changing information on origin,
+seasonality and other factors in the calculation ([more Informations on the indicators](http://www.eaternity.org/foodprint/)). It further delivers reports with sophisticated and easy to understand statistics, graphs and analysis ([more Informations on the reports](http://www.eaternity.org/app/report)).
```

Comment:
"information" should not be plural -- in English it is treated as a substance, like water.